update/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,41 @@
+// VSCode devcontainer configuration. See
```

Comment:
As discussed, I believe the `devcontainer.json` in root is very outdated and it's not working.
I'd opt for removing it instead, since no one mentioned it's being used.